modules/artifact/src/main/java/org/mule/runtime/module/artifact/classloader/FilteringArtifactClassLoader.java

```diff
@@ -112,6 +112,11 @@ else if (logger.isTraceEnabled())
         }
     }
 
+    private Boolean isVerboseClassLoading()
```

Comment:
Didn't find a good place to set this, there are a few similar cases. Perhaps this methods could be added to MuleProperties class in addition to constants.